.github/workflows/release.yaml

```
@@ -75,7 +75,7 @@ jobs:
           tag_name: "${{ env.ref_name }}"
           skip_existing: true
       - name: update to emqx.io
-        if: startsWith(env.ref_name, 'v') && ((github.event_name == 'release' && !github.event.release.prerelease) || inputs.publish_release_artefacts)
+        if: github.event_name == 'release' || inputs.publish_release_artefacts
```

Not introduced by this PR, but:

```suggestion
        if: github.event_name == 'release' || inputs.publish_release_artifacts
```